entrypoint.sh

```diff
@@ -44,6 +44,7 @@ elif [ -n "$POSTGRES_PORT_5432_TCP" ]; then
 		echo >&2 "  Connecting to DB_HOST ($DB_HOST)"
 		echo >&2 "  instead of the linked postgres container."
 	fi
+	: ${DB_NAME:='postgres'}
```

Comment:
- mysql [doesn't come with a default database](https://github.com/docker-library/mysql/blob/b4a2bba171214eadf25aff14a13953b64237f5d8/5.6/docker-entrypoint.sh#L70) that it connects to
- postgres does, and it's called "postgres". [You can manually change to another name, but 'postgres' will still be available](https://github.com/docker-library/postgres/blob/master/9.3/docker-entrypoint.sh#L45)

Ease of use dictates that if there's a default database which gets connected to, barring user-modified settings, that drupal should connect to and use that.